tools/pipelines/templates/include-publish-npm-package-steps.yml

```diff
@@ -70,6 +72,14 @@ steps:
       --retry $maximumRetryIfNetworkError \
       --verbose
 
+      flub promote package \
+      --version "$(SetVersion.version)" \
+      --orderFile $orderFile \
+      --token ":$ADO_API_TOKEN" \
```

Comment:
You don't want to pass tokens using the flag - you want to use the env variable. So just remove this.

Reply:
Maybe with a comment that the value is ready from the env variable.